packages/suite-desktop/src-electron/libs/processes/TorProcess.ts

```diff
@@ -55,6 +55,7 @@ class TorProcess extends BaseProcess {
 
     async start(): Promise<void> {
         const controlAuthCookiePath = path.join(this.authFilePath, 'control_auth_cookie');
+        const electronProcessId = process.pid;
```

Comment:
Why is it called `electronProcessId`? It is `torProcessId`, isnt it? Or do I miss something?

Reply:
It is actually electron process, because it is the process that starts Tor. Tor will be monitoring that process is running and if it is not running it will kill it self. 

The reason is to avoid Tor process being running when suit is not running.